.changeset/thick-ladybugs-tie.md

```diff
@@ -0,0 +1,5 @@
+---
+"@globalfishingwatchapp/fishing-map": patch
+---
+
+Allow GFW users to download vessel tracks
```

Comment:
A changeset is only needed when there is a dependency change, the app doesn't need it

Reply:
I don't know how to remove it :/